akka-actor-typed/src/main/scala/akka/actor/typed/internal/DelayedStart.scala

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (C) 2009-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.actor.typed.internal
+
+import akka.actor.typed.{ ActorContext, Behavior, Signal, scaladsl }
+import akka.annotation.InternalApi
+
+/**
+ * INTERNAL API
+ *
+ * A behavior that wraps another behavior, stashing all incoming messages and signals until
+ * it is sent the `Start` message, it then unstashes all messages and signals and switches to the delayedBehavior
+ */
+@InternalApi
+private[akka] object DelayedStart {
+
+  case object Start
+
+  // note that we cannot use StashBuffer here as we need to buffer signals as well as messages
```

Comment:
I think you can use a StashBuffer[Any], also for signals

Reply:
use the `MutableStashBuffer` since I think I will remove the immutable

Reply:
Ok will take another look at if I can use that

Reply:
obsolete comment now